src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -55,7 +55,7 @@ export const uiSchema = {
               input => input.replace(/([^a-zA-Z0-9\-’.,/() ]+)/g, ''),
               // Get rid of extra whitespace characters
               input => input.trim(),
-              input => input.replace(/\s{2,}/, ' '),
+              input => input.replace(/\s{2,}/g, ' '),
```

Comment:
Completely unrelated change, but I just noticed this tiny 'bug' on the 'Add Disabilities' autosuggest field and didn't feel like opening a new PR for it 😛

Reply:
Good catch!